packages/manager/apps/web-ongoing-operations/src/hooks/data/data.tsx

```diff
@@ -26,3 +27,10 @@ export const useDomainArgument = (id: number, argumentType: string) => {
     queryFn: () => getmeTaskDomainArgument(id, argumentType),
   });
 };
+
+export const useNicList = (id: number) => {
+  return useQuery<string[]>({
+    queryKey: [id],
+    queryFn: () => getmeTaskDomainNicList(id),
```

Comment:
```suggestion
    queryFn: () => getmeTaskDomainArgument(id),
```

function `getmeTaskDomainNicList` and `getmeTaskDomainArgument`are wrongly named